src/pages/WorkloadDetails/WorkloadInfo/WorkloadServices.tsx

```diff
@@ -141,7 +131,7 @@ class WorkloadServices extends React.Component<WorkloadServicesProps, WorkloadSe
         name: this.overviewLink(service),
         createdAt: <LocalTime time={service.createdAt} />,
         type: service.type,
-        labels: this.renderLabels(service.labels || {}, vsIdx),
+        labels: <Labels key={'pod_' + vsIdx} labels={service.labels} />,
         resourceVersion: service.resourceVersion,
```

Comment:
service.labels could be not defined 
```typescript
export interface Service {
  name: string;
  createdAt: string;
  resourceVersion: string;
  namespace: Namespace;
  labels?: { [key: string]: string };
  type: string;
  ip: string;
  ports?: Port[];
}
```
set  ||  {}

Reply:
Good point! I allowed `<Labels />` component to take empty labels. In that case, we show 'No Labels' literal.